File: client/iih_stub/iih.ci_stub/iih.ci.mr/relapredto/d/RelaPreDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.relapredto.d
{
    /// <summary>
    /// RelaPreDTO 的摘要说明。
    /// </summary>
    public class RelaPreDTO : BaseDTO {

        public RelaPreDTO() {
 
        }

        /// <summary>
        /// 医嘱号
        /// </summary>
		public string Id_pre {
            get { return getAttrVal<string>("Id_pre",null); }
            set { setAttrVal<string>("Id_pre", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_pre"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.relapredto.d.RelaPreDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/relapredto/d/RelaPreDTO.java
package iih.ci.mr.relapredto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 关联医嘱信息DTO DTO数据 
 * 
 */
public class RelaPreDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 医嘱号
	 * @return String
	 */
	public String getId_pre() {
		return ((String) getAttrVal("Id_pre"));
	}
	/**
	 * 医嘱号
	 * @param Id_pre
	 */
	public void setId_pre(String Id_pre) {
		setAttrVal("Id_pre", Id_pre);
	}
}